admin/src/css/theme.less

```diff
@@ -13,7 +13,19 @@ html {
   background-color: #323232;
   height: 64px;
   display: flex;
+
+  .extras {
+    float: right;
+    padding: 20px;
+
+    a{
+      color: #DDDDDD;
+      width: 40px;
```

Comment:
I think we should remove the width here and let it expand as much as it needs to to fit the text.